### ISSUE TITLE
[Akinator] Error

### ISSUE THREAD

**Issue by Merp-Me** [Predeactor/Predeactor-Cogs#43]
## Bug description:
Wen running Akinator, the bot stops responding at a random time, appears as typing and then returns with 
`Error in command 'akinator start'. Check your console or logs for details.`

### Steps to reproduce:
Been random so far so unsure at the moment. If i figure it out, I'll update this message.

### Expected Behaviour:
Bot should carry on until the end of the game
### Versions and infos:
 - Red Version: 3.4.9
 - Discord.py Version: 1.7.1

## Logs/Traceback:

```py
Exception in command 'akinator start'
Traceback (most recent call last):
  File "/home/pi/redenv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/pi/.local/share/Red-DiscordBot/data/test/cogs/CogManager/cogs/akinatorcog/akinatorcog.py", line 99, in start
    await self.game_with(ctx, locale)
  File "/home/pi/.local/share/Red-DiscordBot/data/test/cogs/CogManager/cogs/akinatorcog/akinatorcog.py", line 129, in game_with
    question_count = await self.continuous_question(
  File "/home/pi/.local/share/Red-DiscordBot/data/test/cogs/CogManager/cogs/akinatorcog/akinatorcog.py", line 181, in continuous_question
    next_question = await akinator.answer(user_input)
  File "/home/pi/.local/share/Red-DiscordBot/data/test/cogs/Downloader/lib/akinator/async_aki/async_akinator.py", line 189, in answer
    resp = self._parse_response(await w.text())
  File "/home/pi/.local/share/Red-DiscordBot/data/test/cogs/Downloader/lib/akinator/async_aki/async_akinator.py", line 88, in _parse_response
    return json.loads(",".join(response.split("(")[1::])[:-1])
  File "/home/pi/.pyenv/versions/3.8.6/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/home/pi/.pyenv/versions/3.8.6/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/pi/.pyenv/versions/3.8.6/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pi/redenv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 939, in invoke
    await ctx.command.invoke(ctx)
```
```py

  File "/home/pi/redenv/lib/python3.8/site-packages/redbot/core/commands/commands.py", line 832, in invoke
    await super().invoke(ctx)
  File "/home/pi/redenv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 1348, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "/home/pi/redenv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 863, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "/home/pi/redenv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 94, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by Predeactor** [comment]
This is a library issue but that still can be tried to be handled by the cog.
May I ask what was the latest input to the bot before Akinator crash?

**Comment by Merp-Me** [comment]
It was just multiple games of akinator that did were working properly.

**Comment by Predeactor** [comment]
May (Or may not:tm:) be fixed in v0.8.3, I'm still letting that here since it's a somewhat dirty solution I've put.

**Comment by Predeactor** [comment]
As the cog is now at the end of life, I'm closing this issue.
